Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetBillingCollectionUITests.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  PaymentSheetBillingCollectionUITests.swift
+//  PaymentSheetUITest
+//
+//  Created by Eduardo Urias on 2/23/23.
+//
+
+import XCTest
+
+final class PaymentSheetBillingCollectionUITests: XCTestCase {
+    var app: XCUIApplication!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        app = XCUIApplication()
+        app.launchEnvironment = ["UITesting": "true"]
+        app.launch()
+    }
+
+    func testCard_AutomaticFields_NoDefaults() throws {
+        loadPlayground(
+            app,
+            settings: [
+                "customer_mode": "guestmode",
+                "currency": "USD",
+                "merchant_country_code": "US",
+                "apple_pay": "off",
+                "automatic_payment_methods": "off",
+                "link": "off",
+                "attach_defaults": "off",
+                "collect_name": "auto",
+                "collect_email": "auto",
+                "collect_phone": "auto",
+                "collect_address": "auto",
+            ]
+        )
+        app.buttons["Checkout (Complete)"].tap()
+
+        guard let card = scroll(collectionView: app.collectionViews.firstMatch, toFindCellWithId: "card") else {
+            XCTFail()
+            return
+        }
+        card.tap()
+        try! fillCardData(app)
+
+        // Complete payment
+        app.buttons["Pay $50.99"].tap()
+        let successText = app.alerts.staticTexts["Success!"]
+        XCTAssertTrue(successText.waitForExistence(timeout: 10.0))
```

Comment:
This is a long timeout. Seems like we should shorten to keep tests snappier when they fail

Reply:
This particular case does need a significant timeout because there is a network request going on, if we make it shorter, say 5 seconds, it might work in most cases, but there is also a chance that it would cause flakiness due to the request taking longer.